src/main/java/org/fundacion/jala/converter/service/AudioConverter.java

```diff
@@ -18,11 +18,31 @@
     private String hz = "";
     private String volume = "";
     private String outputFileName = "";
+    private String audioChannels = "";
     private RunCommand runCommand = new RunCommand();
     private final int ONE_HUNDRED = 1000;
     private final String K20_HERTZ = "20";
     private final String K44_HERTZ = "44";
     private final String K48_HERTZ = "48";
+    private final String MONO_CHANNEL = "1";
+    private final String STEREO_CHANNEL = "2";
+    private final String TWO_POINT_ONE_CHANNEL = "3";
+    private final String FOUR_POINT_ZERO_CHANNEL = "4";
+    private final String FIVE_POINT_ZERO_CHANNEL = "5";
+    private final String FIVE_POINT_ONE_CHANNEL = "6";
+    private final String SIX_POINT_ONE_CHANNEL = "7";
+    private final String SEVEN_POINT_ONE_CHANNEL = "8";
+
+    public AudioConverter() {
+    }
+
+    public AudioConverter(String format, String bitrate, String hz, String volume, String audioChannels) {
```

Comment:
The parameter would be final agree checkstyle

Reply:
solved